docs/changelog.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add default python modules paths to python initialization
 - United config file
 
+### Removed
+- Remove old scp-interpreter
```

Comment:
I propose to write just "Remove scp-interpreter", without "old". Because "new scp-interpreter" isn't declared on this stage

Reply:
Fixed it